client/app/components/packages/pas-form/edit.hbs

```diff
@@ -89,6 +90,7 @@
             </small>
             <Textarea
               @value={{saveable-form.saveableChanges.dcpDescriptionofprojectareageography}}
+              data-test-dcpDescriptionofprojectgeography
```

Comment:
If #333  get merged, this will become `data-test-textarea="dcpDescriptionofprojectareageography"`